.github/workflows/zxcron-promote-build-candidate.yaml

```diff
@@ -230,6 +230,27 @@ jobs:
         with:
           egress-policy: audit
 
+      - name: Report failure (PagerDuty)
+        run: |
+          curl --request 'POST' \
+          --url 'https://events.pagerduty.com/v2/enqueue' \
+          --header 'Content-Type: application/json' \
+          --data '{
+            "payload": {
+              "summary": "Hedera Services - eXtended Test Suite Failure",
+              "severity": "error",
+              "source": "GitHub Actions - Extended Test Suite",
+              "custom_details": {
+                "Fetch Latest Build Candidate": "${{ needs.determine-build-candidate.result }}",
+                "Promote Build Candidate": "${{ needs.promote-build-candidate.result }}",
+                "Source Commit": "<${{ github.server_url }}/${{ github.repository }}/commit/${{ github.sha }}>",
+                "Workflow Run": "<${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}>"
+              }
+            },
+            "routing_key": "${{ secrets.PAGERDUTY_CITR_INTEGRATION_KEY }}",
```

Comment:
Same as above - I don't see this key passed into the workflow. Does it need to be or is it an env token available implicitly to the whole workflow?

Reply:
It's a workflow dispatch trigger and a crib trigger. These have access to the repository secrets implicitly